### PR TITLE
PT-8500: Improvements

### PR DESCRIPTION
**Pull request by akak1977**
## Description
Module rework to resolve tech debt.
### Jira-link:

https://virtocommerce.atlassian.net/browse/PT-8340
https://virtocommerce.atlassian.net/browse/PT-8500

### CODE REVIEW

**Review by ksavosteev — DISMISSED** [review]

**Review by zashchitnik-kuka — DISMISSED** [review]
Language: Fixed and approved

File: docs/index.md [hunk 1]
```diff
@@ -2,260 +2,412 @@
...
+      }
   ]
 }
 ```
+#### Description:
+|Name|Description|
+|-|-|
+|ConnectionName|Name of the connection the event data should be forwarded to.|
+|Name|Human-readable name to distinguish subscriptions. Should be unique in configuration|
+|JsonPathFilter|`JsonPath` filter expression that allows you to additionally filter events that have specific value in the body. If the body with applied `JsonPath` filter does not yield any value, the module will not call the provider. The default value is `$`, which means any event body is OK and may be transferred to the provider.|
```
Comment (OlegoO) [inline docs/index.md]: Add JsonPathFilter examples, for example send event for Specific Order Status Only

File: docs/index.md [hunk 2]
```diff
@@ -2,260 +2,412 @@
...
+|PayloadTransformationTemplate|An optional setting where you can specify a Liquid template to transform event data to a different form. If omitted, null, or an empty string, the event data will be transferred unchanged to the provider (full body).|
```
Comment (OlegoO) [inline docs/index.md]: Add several examples for PayloadTransformationTemplate

File: docs/index.md [hunk 3]
```diff
@@ -2,260 +2,412 @@
...
-Request Body:
 ```json
 {
-  "provider": "AzureEventGrid",
-  "connectionString": "https://demo-demo.eastus-1.eventgrid.azure.net/api/events",
-  "accessKey": "TXSO992J8UQ420HucBgsCs8qiDjFPerCYp0mS6hceM4=",
-  "eventIds": [
-    "VirtoCommerce.CatalogModule.Core.Events.ProductChangedEvent",
-    "VirtoCommerce.CatalogModule.Core.Events.CategoryChangedEvent",
+  "name": "string",
```
Comment (OlegoO) [inline docs/index.md:244]: Change on example with real data

File: docs/index.md [hunk 4]
```diff
@@ -272,7 +424,48 @@ Error:
...
+## Health Status and Searching for Fail Log
+There is API endpoint that allows you to view the fail log.
+
+Endpoint: `/api/eventbus/logs/search`
+
+Method: `POST`
+
+Request:
+
+```json
```
Comment (OlegoO) [inline docs/index.md:437]: Add example how to use for external health monitoring

**Review by OlegoO — CHANGES_REQUESTED** [review]

**Pull request merged**